docs/overview/why-waku.md

```diff
@@ -2,31 +2,23 @@
-
-- It is not tied to a specific chain.
-
-- It is modular and adaptive, enabling various use cases.
-
-- It enables developers to decentralize communication in their dApps or move actions off-chain without compromising decentralization.
-
-- It aims to run anywhere, in the cloud and browser alike.
-
-Private. Secure. Runs anywhere.
-
-|                                        | Whisper                                                                                                       |                                 Waku                                  |
-| -------------------------------------- | ------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------: |
-| **Scalability**                        | Whisper doesn't scale very well, specifically when it comes to bandwidth usage on mobile devices.             |                  Uses gossipsub and content topics.                   |
-| **Spam Resistance**                    | Proof of work requires too much battery and compute power making it a poor mechanism for heterogeneous nodes. |   Uses innovative p2p economic spam protection mechanism RLN Relay    |
-| **No incentivized infrastructure**     | There is no incentive to run a Whisper node                                                                   |  Research in progress to design incentivization for node operators.   |
-| **Formal Specification/Documentation** | Lack of formal and unambiguous specification                                                                  | Open source specs & docs and available for anyone to read or improve. |
-| **Portability**                        | Runs over devp2p which limits where Whisper can run and how                                                   |   Waku is built using libp2p, making it easy to run Waku anywhere.    |
+Communication in the present day is heavily influenced by third-party intervention, ranging from censorship and deplatforming to intermediaries that seek to profit from rent and the misuse of data in the surveillance economy.
+
+Waku is intended to empower individuals by returning control of communication to them. It is the go-to standard for Web3 communication, offering a **scalable decentralized communication** solution.
+
+- Waku improves upon Whisper's capabilities by overcoming limitations and addressing functional gaps.
+- It provides a public infrastructure for the Ethereum and multi-chain ecosystem, serving as a common good.
+- It is not confined to a particular blockchain.
+- It is modular, adaptable, and can cater to various use cases.
+- It allows developers to decentralize communication in their dApps or move actions off-chain while maintaining decentralization.
+- It can run on various platforms, including mobile devices, cloud environments, web browsers, native desktop apps, or even dapp nodes!
+
+## Whisper vs. Waku: Why Waku Wins
```

Comment:
```suggestion
## Whisper vs. Waku: Why Waku is Necessary
```
Just an idea as it's not us vs them but more a "whisper is not good enough and was abandonned"